gradle/dependencies.gradle

```diff
@@ -63,7 +63,7 @@ versions += [
   gradle: "9.1.0",
   grgit: "5.3.3",
   httpclient: "4.5.14",
-  jackson: "2.19.0",
+  jackson: "2.19.2",
```

Comment:
please remember to update the license file, or the CI will fail :smile:

Reply:
Thanks, that's easy to miss. Done.